Defend The Divine/Assets/Scripts/WaveManager.cs

```diff
@@ -4,60 +4,72 @@
 
 public class WaveManager : MonoBehaviour
 {
-    // TODO: Make array
-    public GameObject enemyPrefab;
+    // Can be made into an array if necessary
+    public GameObject[] enemyPrefabs = new GameObject[3];
 
     [SerializeField]
     private Transform[] enemySpawnPositions;
     public Transform[] EnemySpawnPositions { get { return enemySpawnPositions; } }
-
-
-    /* TEMP VARIABLES FOR TEMP ENEMY SPAWNING */
-    //private float spawnDelay = 1f;
-    //private float spawnTimer = 0;
+    
+    [SerializeField]
+    private int currentWave;
+    private const int MinTimeBetweenWaves = 10;
+    private float waveTimestamp = -(MinTimeBetweenWaves/2);
+    
 
     private void Awake() {
         if (enemySpawnPositions.Length == 0) {
-            Debug.LogError("Add enemy spawn positions to array");
+            Debug.LogError("Enemy spawn positions array is empty in WaveManager");
         }
```

Comment:
i like this imma start doing it for my awakes too. Helps us to remember to drag things in